File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& arr, int k) {
        priority_queue<int> qu(arr.begin(), arr.end());
        int ans;
        for(int i = 0;i<k;i++){
            ans = qu.top();
            qu.pop();
        }
        return ans;
        
    }
};

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.cs
public class Solution {
    public int FindKthLargest(int[] arr, int k) {
        PriorityQueue<int, int> q = new PriorityQueue<int, int>();
        for(int i = 0; i < arr.Length; i++){
            q.Enqueue(arr[i], -arr[i]);
        }
        int res = 0;
        while(k > 0){
            res = q.Dequeue();
            k--;
        }
        return res;
    }
}